tests/beaker/tests/create_pv_wo_param_name.rb

```diff
@@ -0,0 +1,37 @@
+require 'master_manipulator'
+require 'lvm_helper'
+test_name "FM-4579 - C96646 - create physical volume without parameter 'name'"
```

Comment:
To make sure physical volume can be created without specifying 'name => "name"' in the manifest, it will take default name instead